server/src/main/java/io/crate/role/metadata/SysUsersTableInfo.java

```diff
@@ -54,6 +56,7 @@ public static SystemTable<Role> create(Supplier<String> clusterId) {
                 .add("role", STRING, GrantedRole::roleName)
                 .add("grantor", STRING, GrantedRole::grantor)
             .endObjectArray()
+            .add("session_settings", UNTYPED_OBJECT, x -> new TreeMap<>(x.sessionSettings()))
```

Comment:
There's a dedicated `addDynamicObject` which I think needs to be used here to get proper subscript support.

Reply:
thx, indeed, it's needed to be able to retrieve individual settings with subscript.